### PR TITLE
Env cleanup

### PR DESCRIPTION
**Pull request by lioramilbaum**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -70,14 +71,15 @@ createDslContainers podName: dslPodName,
             )
           }
         }
-        else {
-          sh 'echo no more than 0'
-        }
+      } catch (Exception e) {
```
Comment (ifireball) [inline Jenkinsfile]: Why not put the cleanup in a finally block?

Comment (lioramilbaum) [reply Jenkinsfile]: @ifireball what is the value in switching catch with finally?

**Comment by lioramilbaum** [comment]
@Akrog please review/merge the PR

File: Jenkinsfile (hunk 1 above)
Comment (ifireball) [reply Jenkinsfile]: `catch` is for handling failure scenarios, you need to run cleanup wither you fail or not, so `finally` is more suitable, also this avoids the need to re-throw the error and make the traceback point to the original error and not the point where it was re-thrown.

**Pull request merged**